conf/dynaconf_hooks.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+from robottelo.logging import logger
+from robottelo.utils.ohsnap import dogfood_repository
+
+# from robottelo import constants  # put filename in the constants? TODO
```

Comment:
I don't think we need to worry about this

Reply:
comment removed